### PR TITLE
 net-im/signal-desktop-bin: bump version 1.11.0 

### PR DESCRIPTION
**Pull request by robert7k**

### CODE REVIEW

File: net-im/signal-desktop-bin/signal-desktop-bin-1.11.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_PN="${PN/-bin/}"
+
+inherit gnome2-utils pax-utils unpacker xdg-utils
+DESCRIPTION="Allows you to send and receive messages of Signal Messenger on your computer"
+HOMEPAGE="https://signal.org/ https://github.com/WhisperSystems/Signal-Desktop"
```
Comment (Zlogene) [inline net-im/signal-desktop-bin/signal-desktop-bin-1.11.0.ebuild]: one hompage per line

File: net-im/signal-desktop-bin/signal-desktop-bin-1.11.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+SRC_URI="https://updates.signal.org/desktop/apt/pool/main/s/${MY_PN}/${MY_PN}_${PV}_amd64.deb"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="ayatana"
```
Comment (Zlogene) [inline net-im/signal-desktop-bin/signal-desktop-bin-1.11.0.ebuild]: why did you add this flag? it does not affect the installation, so install it unconditionally

File: net-im/signal-desktop-bin/signal-desktop-bin-1.11.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+RESTRICT="bindist mirror"
+
+RDEPEND="
+	gnome-base/gconf:2
+	dev-libs/nss
+	x11-libs/libXScrnSaver
+	x11-libs/libXtst
+	net-print/cups
+	ayatana? ( dev-libs/libappindicator:3 )
```
Comment (Zlogene) [inline net-im/signal-desktop-bin/signal-desktop-bin-1.11.0.ebuild]: see above

File: net-im/signal-desktop-bin/signal-desktop-bin-1.11.0.ebuild (hunk 2 above)
Comment (robert7k) [reply net-im/signal-desktop-bin/signal-desktop-bin-1.11.0.ebuild]: See here: https://bugs.gentoo.org/645814

Comment (Zlogene) [reply net-im/signal-desktop-bin/signal-desktop-bin-1.11.0.ebuild]: noted, seems like false positive statement for me

Comment (robert7k) [reply net-im/signal-desktop-bin/signal-desktop-bin-1.11.0.ebuild]: @Zlogene I removed the flag as you suggested.

**Comment by mgorny** [comment]
Thanks!
